src/psyclone/parse/module_manager.py

```diff
@@ -300,7 +443,102 @@ def get_modules_in_file(self, finfo: FileInfo):
 
         return [name.lower() for name in mod_names]
 
-    # ------------------------------------------------------------------------
+    def get_all_recursively_used_module_infos(
```

Comment:
Is this a duplication of the functionality already provided by `get_all_dependencies_recursively` and `sort_modules`?